resources/libopenmp.licence

```diff
@@ -0,0 +1,24 @@
+Copyright (C) 2005, 2007, 2008, 2009 Free Software Foundation, Inc.
```

Comment:
This file is a duplicate of the one in `build/`

Reply:
We should not commit the build/ but this instead, that whole folder is generated from existing resources in the other folders, such as this one.

Reply:
build/  removed